### ISSUE TITLE
MESHROOM 2018 & BLENDER final Texturing issue! 

### ISSUE THREAD

**Issue by potblender** [alicevision/meshroom#1257]
Hi everyone....hope someone can put me on the right path.
Trying Photogrammetry. 

Meshroom 2018... Blender 2.9... WIN7...8GB RAM Python26 (Tried Meshroom 2021 but found out on web to try 2018 due to PC System?)

After initial issues with DepthMap, I've managed to 'finally' get through the full process in Meshroom and to the end of texturing node (though took some time!).

My aim is to create a 3D model of a local statue that stands in a local garden! Up to 55 images taken on a Canon S95 (about 1.6MB each pic)

In the 3D viewer, I can see the outline of  statue (dots of a 3D view) but when I Load Model there is no 3D statue in a 'solid' form and when the Mesh view is ticked it looks like the surrounding garden is around the statue mesh!

![image](https://user-images.githubusercontent.com/78647306/107118776-e3ad0d00-687a-11eb-8544-fe07a4fd2326.png)

![image](https://user-images.githubusercontent.com/78647306/107118798-faebfa80-687a-11eb-8311-c15539af4177.png)

In the MeshroomCache there are the different Node folders with the data etc and in the Texturing folder there is the .mg  .mtl   .obj files... a few png and a couple other files (log/status).

Tried opening the .obj file in Blender and only a 'ring' (of the garden?) shows... (Also tried MeshLab)

![image](https://user-images.githubusercontent.com/78647306/107118740-9a5cbd80-687a-11eb-93e6-c382cb8896b1.png)

Is the problem my system, a glitch in Meshroom, other?

Tried changing settings in different Meshroom nodes to get where I am, but now stuck where I am.
Best settings for the Nodes in Meshroom? Add extra Nodes? Images sizes etc? 
Any help would be most welcome!

**Comment by gpotte** [comment]
I am struggling with the same issues, I am working with big datasets that are not constant so glitching is occuring  lot but the .obj i get is just a small fragment of the reconstructed object in the live viewer.
Interested in a wy to work around this problem

**Comment by natowi** [comment]
- The statue is too dark - use another ISO setting or shoot with daylight
- make sure the statue fills most of the image, so far it makes up only 1/3 of the image
- the surrounding area is well lit and structured and makes up 2/3 of the image, so the algorithm will prefer the surrounding for reconstruction and discard the 1/3 with dark, unstructured area.
- This issue can be fixed by improving the capturing setup - check out the other issues labelled "dataset issues" for tips and tricks

**Comment by potblender** [comment]
Hi Natowi.
I will try and take better close-up photos.
I've used the same images in Zephyr 3D FREE and managed to obtain a mesh (48 out of 50 pics). Looks great!..
...although you can only save the project as a .zep file, meaning it can't be used in another editing program.
I've also tried using COLMAP. The version I've downloaded (3.6) has a couple of .bat files.
When running 'Run Tests.bat' there shows no problems (Green no error linesetc). When running the Colmap.bat the program runs, but after loading the images and going to the first process (Feature Extraction) my laptop just goes sooooo sloooow. Really lagging. Tried changing the graphics card settings (NVIDIA GeForce 540M) but not sure if this is a problem as the settings work on BLENDER?!
Any help in getting COLMAP to run better  would be appreiciated! Thanks again!

**Comment by fabiencastan** [comment]
@potblender Why are you not using Meshroom-2020?
There are more visualization tools in it to analyze the issue. The SfM looks correct, so it sounds really strange to get this completely incorrect mesh.

**Comment by potblender** [comment]
Tried 2020 but had trouble with getting through  DepthMap, and because I'm only on WIN7 I had read 2018 was the way to go.
Seem to get all through the Meshroom nodes now until trying to get a final mesh output to edit on Blender etc

**Comment by potblender** [comment]
Ok, I think it is down to not enough/using too much RAM. 4GB RAM WIN7 GeForceGT540M.
Been running Meshroom through and all looks good until halfway through Texturing.
I ran at default settings with DepthMap Downscale set at 2.
'Abnormal Program Termination : Memory Exhausted' message.
Tried clearing Texturing/MeshFiltering/Meshing nodes and setting Max Points down to 1000000...but again the memory exhausted message!
Current project is 71 images...all checked good!
51 of 71 images seem processed at Texturing.
Is there another setting I could try to get through those remaining 20 images, as in back clearing nodes/change node settings, or would it be best starting again using fewer of those 'good' images.
Any help...again gratefully received!
